src/components/Attachments/AttachmentView/index.js

```diff
@@ -118,6 +119,7 @@ function AttachmentView({
                 onScaleChanged={onScaleChanged}
                 onToggleKeyboard={onToggleKeyboard}
                 onLoadComplete={() => !loadComplete && setLoadComplete(true)}
+                isAttachmentModal={isAttachmentModal}
```

Comment:
We can simplify this and forego the need for the new `PDFPreviewError` component

Instead of passing `isAttachmentModal` prop, we'll just pass `errorLabelStyles` 
```js
errorLabelStyles={isAttachmentModal && [styles.textLabel, styles.textLarge]}
```
And then in `PDFView.index/index.native.js`, we use those styles directly
```js
error={<Text style={this.props.errorLabelStyles}>{this.props.translate('attachmentView.failedToLoadPDF')}</Text>}